yarn.lock

```diff
@@ -7867,6 +8089,20 @@ globby@^6.1.0:
     pify "^2.0.0"
     pinkie-promise "^2.0.0"
 
+globby@^9.2.0:
```

Comment:
Current output of `yarn why globby | grep Found`:

```
=> Found "globby@10.0.2"
=> Found "@liferay/js-insights#globby@11.0.1"
=> Found "@liferay/npm-scripts#globby@11.0.1"
=> Found "mem-fs-editor#globby@9.2.0"
=> Found "@liferay/npm-scripts#liferay-npm-bundler#globby@10.0.2"
=> Found "@liferay/npm-scripts#liferay-npm-bridge-generator#globby@10.0.2"
=> Found "@typescript-eslint/typescript-estree#globby@11.0.1"
=> Found "stylelint#globby@11.0.1"
=> Found "liferay-theme-tasks#globby@4.1.0"
=> Found "metal-tools-soy#globby@6.1.0"
=> Found "react-dev-utils#globby@8.0.2"
=> Found "del#globby@6.1.0"
=> Found "yeoman-generator#globby@8.0.2"
=> Found "yeoman-environment#globby@8.0.2"
=> Found "yeoman-environment#yeoman-generator#globby@9.2.0"
```

🤦‍♂️

I'll see if I can do a wee bit of de-duping as I get the workspaces properly configured (in another PR), but realistically, we are going to be stuck with a few versions of this darn library no matter what we do.